### ISSUE TITLE
Docker image is named 'latest' and it should be 'openspeedtest'

### ISSUE THREAD

**Issue by JoeNyland** [openspeedtest/Docker-Image#30]
Docker images are usually referenced as `org/image:tag`, with `org/image:latest` being the latest version of the image. For example, if Google were to have an image called Gmail, then the image would be named `google/gmail`.

In OpenSpeedTest's case, the image is currently called `latest`, which doesn't make sense. It implies that the application is called 'latest', when in actual fact, the application is called 'openspeedtest'.

This caught me out and I'm an experienced Docker user. I'm sure it's caught others out, too (for example: https://github.com/openspeedtest/Docker-Image/issues/18) and especially those who are just getting started with Docker.

It gets even more confusing when someone wants to lock their deployment to a specific version. With the current image name, they would:

```
docker run ... openspeedtest/latest:v2.0.5 # Is is the latest version I'm running here? Or is it v2.0.5?
```

Looking at that command, it looks like I'm running the latest version, but I'm not...

The standard solution is to name the image `openspeedtest/openspeedtest`, then to users would run which version they wanted to in an expected way:

```
docker run ... openspeedtest/openspeedtest
docker run ... openspeedtest/openspeedtest:latest
docker run ... openspeedtest/openspeedtest:v2.0.5
```

I understand that it's suboptimal having 'openspeedtest' essentially duplicated, but this is pretty common in the open source world of Docker images.

**Comment by openspeedtest** [comment]
Yeah, that’s true. I’m working on a huge update and will be back soon.

**Comment by openspeedtest** [comment]
to lock image use docker run ... openspeedtest/openspeedtest:v2.0.5
now updated the base image, so docker run ... openspeedtest/openspeedtest:v2.0.6

**Comment by JoeNyland** [comment]
I'm not sure what you're suggesting we do with the last comment. The image `openspeedtest/openspeedtest` doesn't exist (on Docker Hub at least).

```
$ docker pull openspeedtest/openspeedtest:v2.0.5
Error response from daemon: pull access denied for openspeedtest/openspeedtest, repository does not exist or may require 'docker login'
```

**Comment by openspeedtest** [comment]
sorry, it is `openspeedtest/latest:v2.0.6`

**Comment by sellers** [comment]
I think you missed the point of @JoeNyland 's post.  
The path for your docker image is openspeedtest which is great, aligns with your github repository name which is ideal for authenticity/intuition. 

The image name you have is called latest.  The image tag (version) is called 2.0.6.  So, to reverse that, i'm installing version "2.0.6" of the application "latest" from the docker group "openspeedtest" .  


This would be the same as what @JoeNyland  was saying, I'm installing version "2.0.6" of the application "latest" from the docker group "google".  See the problem.  In this example (fake) Google produces Gmail, Photos, Chat, Kaniko, etc however that information is lost/overloaded. 

Recommendation: 

in your ci/cd (github action/gitlab pipeline) have a docker build (or similar) tag the image as openspeedtest:${semantic-version} and then also as openspeedtest:latest.  Then, docker push it to Dockerhub under the 'openspeedtest' group and it will turn into 
`openspeedtest/openspeedtest:latest` or `openspeedtest/openspeedtest:2.0.6` 

I hope that helps explain what the O.P. was trying to accomplish.  

Best wishes.
